modules/keypad/commerce_pos_keypad.install

```diff
@@ -0,0 +1,214 @@
+<?php
+
+/**
+ * @file
+ * Schema and install hooks for commerce_pos.
+ */
+
+/**
+ * Implements hook_schema().
+ */
+function commerce_pos_keypad_schema() {
+  $schema['commerce_pos_currency_denomination_type'] = array(

```

Comment:
It seems to me like this whole file should be deleted. Perhaps you started out with changing this module but decided to break the new functionality into a new module and left this file?